### PR TITLE
SCLD-12577: Fix update server URLs to work for MSI not exe

### PR DESCRIPTION
**Pull request by breen-laerdal**
# SCLD-12577

The update server was not working properly for MSI builds. There were various places where it was looking for `exe` files, which was what we used to use when we built with Squirrel.

## Testing

1. Try a few different endpoints on the preview site in the comment below:2. 
Get Update:
```sh
curl --request GET \
  --url https://sc-cloud-trayapp-updater-git-fix-update-server-laerdal-labs.vercel.app/update/win32/2.2.1
```
Download Update:
```sh
curl --request GET \
  --url 'https://sc-cloud-trayapp-updater-git-fix-update-server-laerdal-labs.vercel.app/download/win32?update=true'
```

Make sure that both endpoints succeed, that the first endpoint returns a 202 with a URL in it and the second one returns the installer itself.

Point the same endpoints at production (https://trayapp-updater.simcapture.com) and make sure that the first one fails with a 404 and the second one returns a 204.

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ module.exports = ({ cache, config }) => {
     } else if (userAgent.isMac && !isUpdate) {
       platform = 'dmg'
     } else if (userAgent.isWindows) {
-      platform = 'exe'
+      platform = 'msi'
```
Comment (breen-laerdal) [inline lib/routes.js:51]: This is the only actual change in here - everything else is auto-formatting changes.

File: lib/aliases.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const aliases = {
   darwin: ['mac', 'macos', 'osx'],
-  exe: ['win32', 'windows', 'win'],
+  msi: ['win32', 'windows', 'win'],
```
Comment (breen-laerdal) [inline lib/aliases.js:3]: This is the only actual change in this file - everything else is formatting

File: package.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
   "main": "lib/index.js",
   "description": "Lightweight update server for Electron apps",
   "scripts": {
-    "dev": "npx vercel dev",
```
Comment (breen-laerdal) [inline package.json:7]: Newer versions of the vercel CLI don't allow you to recursively call `dev` from `dev`, so I renamed this to `local`

File: package.json [hunk 4]
```diff
@@ -4,8 +4,8 @@
...
-    "test": "xo && jest",
+    "local": "npx vercel dev",
+    "test": "xo && jest --passWithNoTests",
```
Comment (breen-laerdal) [inline package.json:8]: Tests were failing without this

**Comment by herereadthis** [comment]
https://sc-cloud-trayapp-updater-git-fix-update-server-laerdal-labs.vercel.app/download/win32?update=true  should prompt a `.msi` download

**Review by herereadthis — APPROVED** [review]

**Pull request merged**